example/args.cpp

```diff
@@ -0,0 +1,10 @@
+#include <process>
```

Comment:
just want to clarify -- none of the example code is pulled into the paper as far as I can see?  if no, I'll work on doing that.

Reply:
They are the boos.process examples just modified.

Reply:
ok great